lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -538,6 +1020,25 @@ module Resources
               'code': '1152',
               'display': 'Emergency Contact'
             }
+          ],
+          'extension': [
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'code': '156'
+                  },
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
```

Comment:
I think we only support the proprietary codes for the create capability.  Can we also switch to using the code 155 for father since the name of the RelatedPerson in this example is John.

Reply:
removed standard code here https://github.com/cerner/fhir.cerner.com/pull/644/commits/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL1039-L1042
added 155 here https://github.com/cerner/fhir.cerner.com/pull/644/commits/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR1048

Reply:
+1